tests/test.cpp

```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+#include <thread>
+
+int main() {
+    std::jthread t([] { std::cout << "Hello, jthread!" << std::endl; });
+    return 0;
+}
```

Comment:
Not needed, right?

Reply:
Made all the changes!

Reply:
Thank you

Reply:
Still outstanding, the file is still present 😔